intellij-toml/core/src/main/kotlin/org/toml/ide/annotator/TomlAnnotator.kt

```diff
@@ -24,5 +31,24 @@ class TomlAnnotator : AnnotatorBase() {
                 holder.newAnnotation(HighlightSeverity.ERROR, "Inline tables are intended to appear on a single line").create()
             }
         }
+
+        val parent = element.parent
+        if (element.elementType == TomlElementTypes.COMMA && parent is TomlInlineTable &&
+            element.textOffset > parent.entries.lastOrNull()?.textOffset ?: 0) {
+            val message = "Remove trailing comma"
+
+            val fix = object : LocalQuickFix {
+                override fun getFamilyName(): String = message
+                override fun applyFix(project: Project, descriptor: ProblemDescriptor) = descriptor.psiElement.delete()
```

Comment:
```suggestion
                override fun applyFix(project: Project, descriptor: ProblemDescriptor) {
                    descriptor.psiElement?.delete()
                }
```
`descriptor.psiElement` can actually return null, so let's avoid NPE here